lib/mergometer/pull_request.rb

```diff
@@ -61,7 +67,7 @@ def heavily_commented?
     end
 
     def comment_count
-      @_comment_count ||= data.comments + comment_data.size
+      @comment_count ||= data.comments + comment_data.size
```

Comment:
We have for convention to prefix memoize variable with a `_`. Do you have any justification for the change? (Same with `@_reviewers` and the others below.)

Reply:
Use the convention which is used by  Ruby community guidelines as you suggested.